### DISCUSSION TITLE
"react-hook-form": "^7.17.4"  problem with getFieldState

### DISCUSSION THREAD

**Discussion by ArtemFedorchuk** [react-hook-form/react-hook-form#7891] [Q&A] [answered]
I use "react-hook-form": "^7.17.4"
          "typescript": "^4.4.3",
But if I want to use getFieldState from the useForm I have error message (TS2339: Property 'getFieldState' does not exist on type 'UseFormReturn '.)
I checked form.d.ts in node_modules I didn't find such a method

**Comment by Moshyfawn** [comment, marked as answer]
The `getFieldState` API was introduced in [Version 7.25.0](https://github.com/react-hook-form/react-hook-form/releases/tag/v7.25.0)

**Reply by majidali129** [reply]
what should we do now?
